lib/lambda_ethereum_consensus/p2p/gossip/operations_collector.ex

```diff
@@ -82,40 +87,17 @@ defmodule LambdaEthereumConsensus.P2P.Gossip.OperationsCollector do
       attestations: block.body.attestations
     }
 
-    GenServer.cast(__MODULE__, {:new_block, block.slot, operations})
+    filter_messages(block.slot, operations)
```

Comment:
If this is the only place where we call filter_messages, let's move the contents here directly, and we can avoid constructing the "operations" dictionary. We can use the block directly.